x/staking/types/staking.pb.go

```diff
@@ -1211,590 +1211,582 @@ func (this *Pool) Description() (desc *github_com_gogo_protobuf_protoc_gen_gogo_
 func StakingDescription() (desc *github_com_gogo_protobuf_protoc_gen_gogo_descriptor.FileDescriptorSet) {
```

Comment:
Why does this file keep changing?

Reply:
no idea but probably something to dig more into.